wp-web-push/wp-web-push-admin.php

```diff
@@ -74,11 +52,10 @@ public function options() {
         wp_die(__('Invalid value for `Registration Behavior`', 'wpwebpush'));
       }
 
-      $triggers_option = array_key_exists('webpush_triggers', $_POST) ? $_POST['webpush_triggers'] : array();
-      $allowed_trigger_values = $this->array_values_recursive($ALLOWED_TRIGGERS);
-      foreach ($triggers_option as $trigger_option) {
-        if (!in_array($trigger_option, $allowed_trigger_values)) {
-          wp_die(sprintf(__('Invalid value in Push Triggers: %s', 'wpwebpush'), $trigger_option));
+      $triggers_option = $_POST['webpush_triggers'];
```

Comment:
IIRC the warning was printed when you tried to change the settings, when you didn't select any of the trigger options.